pdns/dnsdist-lua.cc

```diff
@@ -1967,13 +1967,7 @@ vector<std::function<void(void)>> setupLua(bool client, const std::string& confi
   setupLuaRules();
   setupLuaVars();
 
-  std::ifstream ifs(config);
-  if(!ifs)
-    warnlog("Unable to read configuration from '%s'", config);
```

Comment:
Not being able to open the configuration file used to be only a warning (which may or may not have been a good idea) but will now prevent dnsdist from starting, correct?

Reply:
Yes, it's fatal now - it probably should be

Reply:
I use `-l 127.0.0.1:5300 127.0.0.1:5301` to quickly spawn `dnsdist` to test things, this would make life a bit harder (having to create a config file)

Reply:
right

Reply:
unless `-C /dev/null` also works :)

Reply:
> unless `-C /dev/null` also works :)

Haha, I came here to post this! Yes, that works.

Reply:
@rgacogne @pieterlexis with that addition, is the change acceptable?

Reply:
I really would like dnsdist to keep starting even if it couldn't read its configuration file, at least for 1.4.0. Perhaps we could still try to open it and only try to parse it that succeeded?
I'm OK with changing that behavior for 1.5.0.